connect/runtime/src/test/java/org/apache/kafka/connect/runtime/standalone/StandaloneHerderTest.java

```diff
@@ -630,12 +641,6 @@ private void expectDestroy() {
         return generatedTaskProps;
     }
 
-
-    private void expectConfigValidation(Map<String, String> ... configs) {
-        Connector connectorMock = PowerMock.createMock(Connector.class);
-        expectConfigValidation(connectorMock, true, configs);
-    }
```

Comment:
This signature without an explicit connector mock was a source of most of the failing tests due to `Connector.class` being used for the default mock, as @ewecp suggested. I'm removing this variant in favor of always explicitly passing a connectorMock that uses the appropriate subclass.